Assets/Scripts/Model/Players/EasyAI.cs

```diff
@@ -8,21 +8,21 @@ public class EasyAI : AI
 	public override void Play(){
         InvokeRepeating("autoCoins", 0.0f, 1.0f);
         //create 20 units every wave
-	    Shop shop = GameObject.FindGameObjectWithTag("Shop").GetComponent<Shop>();
+	    UnitShop shop = GameObject.FindGameObjectWithTag("Shop").GetComponent<UnitShop>();
         if (unitsWave < 20)
```

Comment:
This was @JoanBenito 's thing, to say that a wave finishes after the AI spawned 20 units. It's hardcoded, shouldn't be like this. I guess @JoanBenito 'll fix this